.circleci/config.yml

```
@@ -22,20 +22,21 @@ jobs:
             - ~/.cache
       - run: npm run build
       - run: npm test
-      - run:
-          name: npm run test:kitchen-sink
+      - run: 
+          name: browserstack
           command: |
             if [ -z "$CIRCLE_PR_NUMBER" ]; then
-              npm run test:kitchen-sink
+              npx start-server-and-test 'start:kitchen-sink-local' http-get://127.0.0.1:3000 'browserstack-cypress run --build-name $CIRCLE_BRANCH'
```

```suggestion
              npx start-server-and-test 'start:kitchen-sink-local' http://127.0.0.1:3000 'browserstack-cypress run --build-name $CIRCLE_BRANCH'
```

Just want to see if this fixes the FF flakey test (I doubt it will, but just experimenting)